azurerm/resource_arm_container_group.go

```diff
@@ -169,9 +169,19 @@ func resourceArmContainerGroup() *schema.Resource {
 						},
 
 						"command": {
-							Type:     schema.TypeString,
+							Type:       schema.TypeString,
+							Optional:   true,
+							ForceNew:   true,
+							Computed:   true,
+							Deprecated: "Use `commands` instead.",
```

Comment:
Should these have `ConflictsWith`?

Reply:
Should - but it’s an unsized list so that’s not possible (and customizeDiff doesn’t handle interpolations, which could be used here - so I don’t see that as a viable approach)